### PR TITLE
Added information about pulling container images

### PR DESCRIPTION
**Pull request by ianf77**
Added admonition and tidied the content.

Update document to reflect new Red Hat registry network changes

https://issues.redhat.com/browse/AAP-9698

### CODE REVIEW

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 1]
```diff
@@ -261,5 +261,22 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
...
+|link:https://registry.redhat.io:443[https://registry.redhat.io:443] |Access to container images provided by Red Hat and partners
+| `cdn.quay.io:443` | Access to container images provided by Red Hat and partners
+| `cdn01.quay.io:443` | Access to container images provided by Red Hat and partners
+| `cdn02.quay.io:443` | Access to container images provided by Red Hat and partners
+| `cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
 |===
+
+[IMPORTANT]
+====
+Image manifests and filesystem blobs are served directly from `registry.redhat.io` and `registry.access.redhat.com`.
```
Comment (ericzolf) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:273]:
```suggestion
Image manifests and filesystem blobs are served directly from `registry.redhat.io`. 
```
As we don't seem to have it anywhere, let's avoid confusion.

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 2]
```diff
@@ -261,5 +261,22 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
...
+| `cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
 |===
+
+[IMPORTANT]
+====
+Image manifests and filesystem blobs are served directly from `registry.redhat.io` and `registry.access.redhat.com`. 
+However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 
+To avoid problems pulling container images, you must enable outbound connections to the listed `quay.io` hostnames.
+
+This change should be made to any firewall configuration that specifically enables outbound connections to `registry.redhat.io` or `registry.access.redhat.com`.
```
Comment (ericzolf) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:277]:
```suggestion
This change should be made to any firewall configuration that specifically enables outbound connections to `registry.redhat.io`.
```

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 3]
```diff
@@ -261,5 +261,22 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
...
+
+Use the hostnames instead of IP addresses when configuring firewall rules. 
+
+After making this change you can continue to pull images from `registry.redhat.io` and `registry.access.redhat.com`. You do not require a `quay.io` login, or need to interact with the `quay.io` registry directly in any way to continue pulling Red Hat container images.
```
Comment (ericzolf) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:281]:
```suggestion
After making this change you can continue to pull images from `registry.redhat.io`.
You do not require a `quay.io` login, or need to interact with the `quay.io` registry directly in any way to continue pulling Red Hat container images.
```

Should there be also a link to https://access.redhat.com/articles/6999582 for more information? The article is now published.

**Comment by hherbly** [comment]
LGTM!

**Pull request merged**